csi.proto

```diff
@@ -576,8 +576,9 @@ message ControllerPublishVolumeRequest {
   // This is a REQUIRED field.
   VolumeCapability volume_capability = 3;
 
-  // Whether to publish the volume in readonly mode. This field is
-  // REQUIRED.
+  // Whether to publish the volume in readonly mode. This field MUST be
+  // specified only if SP has PUBLISH_READONLY controller capability.
```

Comment:
`This field MUST be REQUIRED specified only if` conflicts with `This is a REQUIRED field`.

We should say that if the SP does not have PUBLISH_READONLY capability, the CO will set this field to `false`.